src/backend/distributed/executor/multi_executor.c

```diff
@@ -66,7 +66,8 @@ static CustomExecMethods RouterSingleModifyCustomExecMethods = {
 	.ExplainCustomScan = CitusExplainScan
 };
 
-static CustomExecMethods RouterMultiModifyCustomExecMethods = {
+/* not static to enable swapping in multi-modify logic during router execution */
+CustomExecMethods RouterMultiModifyCustomExecMethods = {
```

Comment:
Err, misunderstood what you meant with "swapping in multi-modify logic" - thought you were replacing an individual callback in it. Sorry for the noise.  Why not just say 'so it can be referenced by the multi-modify' logic?